charts/osm-arc/values.yaml

```diff
@@ -12,12 +12,12 @@ OpenServiceMesh:
   testedDistros: "aks gke eks openshift rancher_rke"
 
 osm:
-  OpenServiceMesh:
+  osm:
```

Comment:
this will mean users will have to set values as ex: `osm.osm.enableDebugSever` in configuration settings inputs...may seem a little odd or confusing? We can address this in a separate PR but ideally, if it's ok on the Arc side, I think we should change the top level title

Reply:
I don't think there's a way that we can change this on the arc side. As long as the upstream chart is named "osm," and its values.yaml has the "osm" heading, then we would have to use `osm.osm.`

Chart name (published on github pages): https://openservicemesh.github.io/osm/
Values.yaml: https://github.com/openservicemesh/osm/blob/v1.0.0-rc.4/charts/osm/values.yaml#L5

Reply:
@nshankar13 is the [top level](https://github.com/Azure/osm-azure/blob/main/charts/osm-arc/values.yaml#L14) osm key designated by arc?